sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -13,9 +13,15 @@ struct<multiplydtinterval(subtracttimestamps(TIMESTAMP '2019-10-15 10:11:12.0010
 -- !query
```

Comment:
shall we run `interval.sql` one more time with legacy config enabled? by using `--IMPORT`